vuepress/docs/docs/schema.md

```diff
@@ -171,3 +171,23 @@ You can also use the `scalarSchema` helper to create your own scalar types, prov
 import caliban.schema._
 implicit val unitSchema: Schema[Any, Unit] = scalarSchema("Unit", None, _ => ObjectValue(Nil))
 ```
+
+## Schema migration
+
+Caliban can automatically generate Scala code out of a GraphQL schema provided as a json file for easier and faster development. 
+
+In order to use this feature add the caliban sbt plugin to your project. 
+ 
+```scala
+addSbtPlugin("com.github.ghostdogpr" % "caliban-codegen" % "0.5.0")
```

Comment:
I'm going to release 0.5.0 now but don't worry about it, I will update the version myself after each new release.